src/EventSubscriber/OgEventSubscriber.php

```diff
@@ -69,7 +69,11 @@ public function provideDefaultOgPermissions(PermissionEventInterface $event) {
    *   The default role event.
    */
   public function provideDefaultRoles(DefaultRoleEventInterface $event) {
-    $event->addRole(OgRoleInterface::ADMINISTRATOR, ['label' => 'Administrator']);
+    $event->addRole([
```

Comment:
I mean if I understand correctly (and correct me if I don't 😉) - I think your goal is to prevent mistakes in the default definition, but it kind of causes double validations - one in OgRole and one in the event.